qml/PingSlider.qml

```diff
@@ -49,8 +48,27 @@ RowLayout {
         visible: valueText === null
         from: root.from
         to: root.to
-        onValueChanged: root.value = value
         stepSize: sliderControl.stepSize
         editable: true
     }
+
+    Binding on value {
+        when: sliderControl.pressed
+        value: sliderControl.value
+    }
```

Comment:
Since this is control specific, should this be inside sliderControl element ?

Reply:
this is control specific, it's taking care of the root control, changing the root.value when sliderControl is pressed.